Parcare/Services/ParkingLotService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Parking.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Parking.Services
+{
+    public class ParkingLotService
+    {
+        private List<ParkingLotModel> _parkedCars;
+        private int _maxSlots;
+        private int _availableSlots;
+
+        public ParkingLotService(int totalSlots)
+        {
+            this._parkedCars = new List<ParkingLotModel>();
+            this._maxSlots = totalSlots;
+            this._availableSlots = _maxSlots - _parkedCars.Count;
+        }
+
+        public bool IsParkingPossible()
+        {
+            return _availableSlots > 0;
+        }
+        public void ParkCar(string carNumber)
+        {
+            if (IsParkingPossible())
+            {
+                this._parkedCars.Add(new ParkingLotModel
```

Comment:
when it is a list, usually the name would be parkedCarList rather than parckedCars, because then in a foreach you might have var parkedCar in parkedCars which is very similar